src/manager/agent.manager.service.ts

```diff
@@ -57,8 +62,8 @@ export class AgentManagerService {
 
         // @tothink move this caching to db
         // adding one second to cache record timeout so that spinDownAgent has time to process before cache deletes the record
-        Logger.info(`record cache limit set to: ${(ttl === 0 ? ttl : ttl + 1000)}`);
-        await this.cache.set(agentId, { containerId, adminPort, httpPort, adminApiKey, ttl }, {ttl: (ttl === 0 ? ttl : ttl + 1000)});
+        Logger.info(`record cache limit set to: ${(ttl === 0 ? ttl : ttl + 1)}`);
```

Comment:
is 1 really enough time?  the point of the ttl + n was to make sure cache didn't get flushed before we had a chance to do what we needed.  perhaps instead of a magic number, this should be env data.

Reply:
Ya long term we probably want an env var, for right now I was just adjusting it since we we're trying to do 1 second, and ttl is in seconds, not milliseconds, so it would be 1 not 1000